src/map/status.h

```diff
@@ -1923,6 +1923,7 @@ enum scb_flag
 	SCB_RANGE   = 0x10000000,
 	SCB_REGEN   = 0x20000000,
 	SCB_DYE     = 0x40000000, // force cloth-dye change to 0 to avoid client crashes.
+	SCB_BODY    = 0x80000000, // Force bodysStyle change to 0
```

Comment:
This is where it gets dangerous. `0x80000000` no longer fits an int32 (but only an uint32). We currently have a mix of signed an unsigned integer for this. Considering it's a bitmask, we can turn it into an uint32 in all places where it's used (for example, `status_calc_*`), but it'll be soon time to change it to 64 bit, since the next value we add will go there.